File: ontrack-web-core/components/widgets/home/PromotionLeadTimeChartWidget.js
import PromotionLevelLeadTimeChart from "@components/promotionLevels/PromotionLevelLeadTimeChart";
import PromotionChartWidget from "@components/widgets/home/PromotionChartWidget";

export default function PromotionLeadTimeChartWidget(props) {
    return (
        <PromotionChartWidget
            prefix="Lead time to"
            chart={PromotionLevelLeadTimeChart}
            {...props}
        />
    )
}

File: ontrack-model/src/main/java/net/nemerosa/ontrack/model/dashboards/widgets/PromotionLeadTimeChartWidget.kt
package net.nemerosa.ontrack.model.dashboards.widgets

import org.springframework.stereotype.Component

@Component
class PromotionLeadTimeChartWidget: AbstractWidget<PromotionChartWidgetConfig>(
    key = "home/PromotionLeadTimeChart",
    name = "Lead time to promotion",
    description = "Chart displaying the lead time to promotion.",
    defaultConfig = PromotionChartWidgetConfig(project = null, branch = null, promotionLevel = null),
    preferredHeight = 30,
)
